src/components/SelectionList/TransactionListItem.tsx

```diff
@@ -202,30 +245,85 @@ function TransactionListItem<TItem extends ListItem>({
                     <>
                         <View style={[styles.flex1, styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween, styles.mb2, styles.gap2]}>
                             <View style={[styles.flexRow, styles.alignItemsCenter, styles.gap1, styles.flex1]}>
-                                <View style={[styles.mw50]}>{userCell(transactionItem.from)}</View>
+                                <View style={[styles.mw50]}>
+                                    <UserCell
+                                        participant={transactionItem.from}
+                                        keyForList={item.keyForList ?? ''}
+                                        showTooltip={false}
+                                        isLargeScreenWidth={false}
+                                    />
+                                </View>
                                 <Icon
                                     src={Expensicons.ArrowRightLong}
                                     width={variables.iconSizeXXSmall}
                                     height={variables.iconSizeXXSmall}
                                     fill={theme.icon}
                                 />
-                                <View style={[styles.flex1, styles.mw50]}>{userCell(transactionItem.to)}</View>
+                                <View style={[styles.mw50]}>
```

Comment:
We need the `styles.flex1` here otherwise the text overflows. I won't block on this though, since it's a simple change we can do in a follow up and it's more important to get this optimization out and resolve conflicts on ongoing PRs.

![Screenshot 2024-05-27 at 12 25 26 PM](https://github.com/Expensify/App/assets/22219519/11bd7bad-c0d6-41f3-a841-f0908269aa3d)